api/models/communitySettings.js

```diff
@@ -59,6 +60,17 @@ export const getCommunitiesSettings = (
           };
         });
       }
+
+      if (data.length > communityIds.length) {
+        return communityIds.map(community => {
```

Comment:
nit: Maybe you can change `.map(community =>` to `.map(communityId` to make this a little more readable, also makes more sense since you're iterating over community ids.